<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportanceFGdialog</class>
 <widget class="QDialog" name="ImportanceFGdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="horizontalSlider01">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider02">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider03">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider08">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider04">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider05">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider06">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider07">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>20</width>
     <height>531</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>30</y>
     <width>1071</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="NameInfolabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Имя ФГ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>891</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>50</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>130</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>170</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>210</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>250</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>290</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>330</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ValuesInfolabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Важность ФГ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetPushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>540</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сброс</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetValuePushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>540</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить изменения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>370</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>410</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider09">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>450</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>490</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>490</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>441</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ExitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider01</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider02</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider03</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider08</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider04</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider05</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider06</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider07</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ResetPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>clickResetValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetValuePushButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setValueImportanceFG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider09</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>735</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider12</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>548</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportanceFGdialog</receiver>
   <slot>setChangeSliders(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setChangeSliders(int)</slot>
  <slot>clickResetValue()</slot>
  <slot>setValueImportanceFG()</slot>
 </slots>
</ui>
